### PR TITLE
Upgrade to spigot 1.11

### PR DESCRIPTION
**Pull request by Aleksey-Terzi**

### CODE REVIEW

File: namelayer-spigot/pom.xml [hunk 1]
```diff
@@ -39,7 +39,7 @@
     <dependency>
       <groupId>vg.civcraft.mc.civmodcore</groupId>
       <artifactId>CivModCore</artifactId>
-      <version>1.5.09</version>
+      <version>1.5.10</version>
```
Comment (ProgrammerDan) [inline namelayer-spigot/pom.xml]: For consistency with latest PR, please update to 1.5.11

File: pom.xml [hunk 2]
```diff
@@ -76,6 +76,7 @@
         <module>nms/v1_8_R3</module>
         <module>nms/v1_9_R1</module>
         <module>nms/v1_9_R2</module>
+		<module>nms/v1_10_R1</module>
```
Comment (ProgrammerDan) [inline pom.xml]: Apparently this xml is still spaces, for now.

File: nms/v1_11_R1/src/main/java/vg/civcraft/mc/namelayer/misc/v1_11_R1/ProfileModifier.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+		if (name.length() > 16) {
+			log.info(String.format("The player %s (%s) was kicked from the server due to his "
+							+ "name already existing but now becoming over 16 characters.",
+					name, player.getUniqueId().toString()));
+		}
+		try {
+			// start of getting the GameProfile
+			CraftHumanEntity craftHuman = (CraftHumanEntity) player;
+			EntityHuman human = craftHuman.getHandle();
+			GameProfile prof = human.getProfile();
```
Comment (ProgrammerDan) [inline nms/v1_11_R1/src/main/java/vg/civcraft/mc/namelayer/misc/v1_11_R1/ProfileModifier.java]: Well this is a welcome improvement; finally a first class Bukkit api call to get the GameProfile? Bravo.

**Comment by ProgrammerDan** [comment]
minor changes then OK to merge

**Comment by Aleksey-Terzi** [comment]
Oh, I was too late with fix for spaces in pom.xml.
I'm guessing this is not the reason to create new pull request :)

**Comment by ProgrammerDan** [comment]
Yes, don't worry about it :)

The style-guide needs to be applied to the project probably, and that will catch the issue anyway. Not worth a separate commit.

**Pull request merged**